quarkus/service/src/main/java/org/apache/polaris/service/quarkus/task/QuarkusTaskExecutorImpl.java

```diff
@@ -39,16 +39,17 @@ public class QuarkusTaskExecutorImpl extends TaskExecutorImpl {
   private final Tracer tracer;
 
   public QuarkusTaskExecutorImpl() {
-    this(null, null, null, null);
+    this(null, null, null, null, null);
   }
 
   @Inject
   public QuarkusTaskExecutorImpl(
+      CallContext callContext,
```

Comment:
The fact that this class is `ApplicationScoped` should mean that it can't be constructed with a `CallContext` -- something is definitely very wrong here